ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/authentication/AuthenticationUtils.java

```diff
@@ -42,6 +53,12 @@ public static boolean isPermitted(
       return true;
     }
 
+    // if the method is configured as a no auth method we skip permission check
+    if (Optional.ofNullable(noAuthMethods).orElse(Collections.emptyList()).stream()
```

Comment:
Can noAuthMethods just be an emptyList in all cases?

Reply:
Makes a lot more sense than what I'm doing here!

Reply:
7d80dd7